### PR TITLE
#10059 change CALL function statement

### PR DESCRIPTION
**Pull request by LonwoLonwo**
2021-04-19 18_21_03-DBeaver Enterprise 21 0 3 - _none_ Script-108

### CODE REVIEW

File: plugins/org.jkiss.dbeaver.ext.db2/src/org/jkiss/dbeaver/ext/db2/DB2SQLDialect.java [hunk 1]
```diff
@@ -106,6 +108,15 @@ protected void loadFunctions(JDBCSession session, JDBCDatabaseMetaData metaData,
         }
     }
 
+    @NotNull
+    @Override
+    protected String getProcedureCallEndClause(DBSProcedure procedure) {
+        if (procedure.getProcedureType() == DBSProcedureType.FUNCTION) {
+            return "FROM SYSIBM.SYSDUMMY1";
```
Comment (ShadelessFox) [inline plugins/org.jkiss.dbeaver.ext.db2/src/org/jkiss/dbeaver/ext/db2/DB2SQLDialect.java:116]: Probably you should leave a comment with a link to the source this goofy table comes from

https://www.ibm.com/docs/en/db2-for-zos/12?topic=tables-sysdummy1

File: plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/impl/sql/AbstractSQLDialect.java [hunk 2]
```diff
@@ -743,6 +743,11 @@ protected String getStoredProcedureCallInitialClause(DBSProcedure proc) {
         }
     }
 
+    @NotNull
+    protected String getProcedureCallEndClause(DBSProcedure procedure) {
+        return "";
+    }
+
```
Comment (ShadelessFox) [inline plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/impl/sql/AbstractSQLDialect.java:750]: Consider moving this method to the super interface [`SQLDialect`](https://github.com/dbeaver/dbeaver/blob/devel/plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/sql/SQLDialect.java)

**Review by ShadelessFox — CHANGES_REQUESTED** [review]

File: plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/impl/sql/AbstractSQLDialect.java (hunk 2 above)
Comment (LonwoLonwo) [reply plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/impl/sql/AbstractSQLDialect.java:750]: I'm not sure about this.
getStoredProcedureCallInitialClause not in SQLDialect.

Comment (ShadelessFox) [reply plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/impl/sql/AbstractSQLDialect.java:750]: Okay, I see.

**Review by ShadelessFox — APPROVED** [review]

**Pull request merged**